src/components/Card/Card.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import Statistic from "../Statistic/Statistic"
+
+class Card extends React.Component {
+
+    constructor(props) {
+        super(props)
+    }
+
+    render() {
+        return (
+            <div className="card">
+                <Statistic type="Breed" details={this.props.name}/>
+                <Statistic type="Height" details={this.props.height}/>
```

Comment:
<img width="302" alt="Screenshot 2019-10-15 at 14 30 41" src="https://user-images.githubusercontent.com/30009848/66836097-a7082480-ef58-11e9-9663-4a064e5ffd84.png">
71 - 86 ....what?

Reply:
on it

Reply:
Looks like this has now been looked at